lighthouse-core/scripts/print-a11y-scoring.js

```diff
@@ -7,9 +7,8 @@
 // node lighthouse-core/scripts/print-a11y-scoring.js
 
 import Config from '../config/config.js';
-import defaultConfig from '../config/default-config.js';
 
-const config = new Config(defaultConfig);
+const config = await Config.createConfigFromJson();
```

Comment:
The defaultConfig is used by default.

Reply:
![image](https://user-images.githubusercontent.com/4071474/167947166-ad334c16-25e9-44c8-83de-c599238de3fb.png)